common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -413,8 +413,7 @@ def check_button_name(self):
         # The logic flow is a little odd so that _('xxx') strings can be found for
         # translation while also running _() just once for each string.
         _ = self.runtime.service(self, "i18n").ugettext
-        check = _('Check')
-        final_check = _('Final Check')
+        check = _('Submit')
 
         # Apply customizations if present
         if 'custom_check' in self.text_customization:
```

Comment:
This customization code (which is now dead) can be removed in https://openedx.atlassian.net/browse/TNL-4939.

Note that the documentation block for check_button_name also needs to be updated. I suppose that can also be done in TNL-4939.